examples/rps-9-web/algoWC.patch

```diff
@@ -4,7 +4,8 @@
  import './index.css';
  import * as backend from './build/index.main.mjs';
  import {loadStdlib} from '@reach-sh/stdlib';
-+import WalletConnect from '@reach-sh/stdlib/ALGO_WalletConnect';
++import * as reachsdk from '@reach-sh/stdlib';
++const WalletConnect = reachsdk.ALGO_WalletConnect;
```

Comment:
See above

Reply:
[b0b8b5d](https://github.com/reach-sh/reach-lang/pull/664/commits/b0b8b5d7c8b6ac33868b17c5f5503189e1ea9701) addresses this.